libr/core/disasm.c

```diff
@@ -4583,6 +4585,17 @@ R_API int r_core_print_disasm(RPrint *p, RCore *core, ut64 addr, ut8 *buf, int l
 				r_cons_printf ("%s%s%s; --------------------------------------\n",
 					COLOR (ds, color_flow), ds->line, COLOR_RESET (ds));
 			}
+		}
+		if (ds->show_seekline && ds->vat == core->prompt_offset) {
+			int pad_len = 2 + R_MAX (0, ds->line ? strlen (ds->line) - 2 : 0);
+			int line_len = R_MAX (0, ds->interactive ? r_cons_singleton ()->columns : 80) - pad_len;
+			r_cons_memset (' ', pad_len);
+			r_cons_print (COLOR_CONST (ds, CYAN));
```

Comment:
honor scr.color and dont hardcode the color, use the theme palette for this

Reply:
can it have its own color in the theme palette?